api/api-facade/api-http/src/main/java/org/eclipse/dirigible/api/v3/http/HttpClientFacade.java

```diff
@@ -195,6 +194,20 @@ private static void prepareHeaders(HttpClientRequestOptions httpClientRequestOpt
 		}
 	}
 
+	private static final String[] supportedTextMimeTypes = new String[] {
```

Comment:
Why not pass the content type strings directly to the `Arrays.asList` method below? This way you won't need a second variable